procrastinate/connector.py

```diff
@@ -7,7 +7,7 @@ class BaseConnector:
     json_dumps: Optional[Callable] = None
     json_loads: Optional[Callable] = None
 
-    async def close_connection(self) -> None:
+    def close_connection(self) -> None:
```

Comment:
Please don't make close a sync function in the abstract class. Asyncpg's close as well as Aiopg's pool close are async, and making it sync will make things quite harder I believe.